setup.py

```diff
@@ -21,8 +19,21 @@
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
     ],
-    description="Multinet API / web application",
-    install_requires=requirements,
+    description="Multinet API",
+    install_requires=[
+        "python-arango",
+        "newick",
+        "uuid",
+        "requests",
+        "webargs",
+        "flask",
+        "flask-cors",
+        "flasgger",
+        "typing-extensions",
+        "uwsgi",
+        "sentry-sdk[flask]",
+        "gunicorn",
+    ],
```

Comment:
Why are these needed here? Since we already have the `requirements.txt`

Reply:
Long story but read this article: https://caremad.io/posts/2013/07/setup-vs-requirement/.

I chatted with David Manthey about it this morning, and the bottom line is, this is the technically correct way to express the "install requirements", and is indeed redundant with the requirements.txt file, but for completeness I did it this way.